pom.xml

```diff
@@ -85,7 +85,7 @@
         <dependency>
             <groupId>org.jboss.aerogear.unifiedpush</groupId>
             <artifactId>unifiedpush-push-model</artifactId>
-            <version>1.1.0.Final</version>
+            <version>1.2.0-SNAPSHOT</version>
```

Comment:
not sure that's needed, especially with the backport to UPS 1.1.x 

I am fine with having the sender on its own version schema... 

I honestly don't know what's best here